modules/cnf-topology-aware-lifecycle-manager-troubleshooting.adoc

```diff
@@ -442,4 +442,13 @@ This may be because:
 * The CGU was run too soon after a policy was created or updated. 
 * The remediation of a policy affects the compliance of subsequent policies in the `ClusterGroupUpgrade` CR.
 
-Resolution:: Create a new and apply `ClusterGroupUpdate` CR with the same specification .
\ No newline at end of file
+Resolution:: Create and apply a new `ClusterGroupUpdate` CR with the same specification.
+
+[discrete]
+=== Clusters deployed with ZTP workflow have no configurations applied
+
+Issue:: The auto-created `ClusterGroupUpgrade` CR in the ZTP workflow has no managed policies and its target cluster has been marked with "ztp-done" label. This means that no policies were available for the target cluster when the cluster became `Ready`. This may be because the `PolicyGenTemplates` were pushed to the git repository before the policies were created.
```

Comment:
Thanks for the update!

This is not accurate, as policies are getting created when pushing policygentemplates to git.
`This may be because the `PolicyGenTemplates` were pushed to the git repository before the policies were created`

Should be something like,
This may be because the `PolicyGenTemplates` were not pushed to the git repository within the desired time window after SiteConfigs were pushed which caused no policies were created when the cluster became Ready.